src/main/java/com/jdbc/App.java

```diff
@@ -0,0 +1,21 @@
+package com.jdbc;
+
+import com.jdbc.dao.UserDao;
+
+import java.sql.*;
+
+public class App {
+    public static void main(String[] args) {
+        UserDao userDao = new UserDao();
+        Connection connection = userDao.getConnection();
```

Comment:
connection should be closed during finding user

Reply:
this feature will be fixed with new class and connection pool